app/javascript/components/Payment/Payment.js

```diff
@@ -421,7 +414,9 @@ export class Payment extends Component {
           {!hideRecurring && (
             <Checkbox
               className="Payment__config"
-              disabled={hideRecurring}
+              disabled={
+                hideRecurring || this.props.currentPaymentType === 'google'
```

Comment:
no recurring payments with Google wallet?

Reply:
Unfortunately not. I read the following in [Braintree's documentation](https://articles.braintreepayments.com/guides/payment-methods/google-pay#recurring-billing-and-vaulting):

> Because each unique Google Pay transaction requires consent from the customer during the checkout flow, vaulting the payment method for future transactions will result in declines and is not recommended.

I also contacted Braintree and they replied that, whilst the token we get back in the response is a valid token for recurring payments, it is not recommended (probably for the reason mentioned above).